yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -20,6 +20,19 @@
 namespace yarpl {
 namespace flowable {
 
+template <typename T>
+class Flowable;
+
+namespace detail {
+
+template <typename T>
+struct is_t_flowable : std::false_type {};
+
+template <typename R>
+struct is_t_flowable<Reference<Flowable<R>>> : std::true_type {};
```

Comment:
Nit: I believe the convention is `is_flowable` for the struct type and `is_flowable_v` for the value type.

Thinking of `std::is_signed` and `std::is_signed_v`.

Reply:
That's for the using alias which is the `::value` of the type; not the type itself

Reply:
```
template <typename T>
using is_t_flowable_v = is_t_flowable<T>::value
```
would be the convention iirc